apps/st2-history/history-popup.component.js

```diff
@@ -59,8 +62,29 @@ export default class HistoryPopup extends React.Component {
   }
 
   handleSubmit(e) {
+    // whenever user changes any parameter, it is stored into payload. So we get changed data into payload. 
+    // We have copy of original data without any parameter change in payloadCopy object on line no 49.
+    // Here we are first identifying key name of secret parameter, payloadKey is key variable name for payload object and payloadCopyKey variable name for payloadCopy object.
```

Comment:
Can you make sure line length is consistent with other lines in this paragraph? Break this into separate lines?